### ISSUE TITLE
Missing title

### ISSUE THREAD

**Issue by abomariam** [fabiocaccamo/django-admin-interface#7]
In this file
 __django-admin-interface/admin_interface/templates/admin/base_site.html__
You forgot to override the title block which caused the admin site title to disappear.

You should add this block to the file
```{% block title %}{{ title }} | {{ site_title|default:_('Django site admin') }}{% endblock %}```

**Comment by fabiocaccamo** [comment]
Each Theme has a "Visible" checkbox under the Title section.

**Comment by abomariam** [comment]
This checkbox is showing the title inside the page body inside the branding block not the head title.

**Comment by fabiocaccamo** [comment]
You are right, I forgot it.

This is how the block title should look like:

```
{% block title %}
{% get_admin_interface_theme as theme %}
{% if theme.title %}{% trans theme.title %}{% else %}{% trans 'Django administration' %}{% endif %}
{% endblock %}
```

I will fix it as soon as possible.
Thanks

**Comment by fabiocaccamo** [comment]
Fixed with 0.1.9, thanks for te report.
